### PR TITLE
[DOC] Improved doc about problems with KafkaRebalance resource

### PR DESCRIPTION
**Pull request by ppatierno**
### Type of change

- Documentation

### Description

This trivial PR tries to explain better what are the common problems with a `KafkaRebalance` resource and how to use the "refresh" annotation for fixing them.
It's related to #3636 

### Checklist

- [x] Update documentation

### CODE REVIEW

**Review by tomncooper — APPROVED** [review]

**Review by scholzj — APPROVED** [review]

File: documentation/modules/cruise-control/proc-fixing-problems-with-kafkarebalance.adoc [hunk 1]
```diff
@@ -9,10 +9,12 @@
 If an issue occurs when creating a `KafkaRebalance` resource or interacting with Cruise Control, the error is reported in the resource status, along with details of how to fix it. 
 The resource also moves to the `NotReady` state.
 
-To continue with the cluster rebalance operation, you must fix the problem in the `KafkaRebalance` resource itself. 
+To continue with the cluster rebalance operation, you must fix the problem in the `KafkaRebalance` resource itself or with the Cruise Control overall deployment.
```
Comment (PaulRMellor) [inline documentation/modules/cruise-control/proc-fixing-problems-with-kafkarebalance.adoc:12]:
```suggestion
To continue with the cluster rebalance operation, you must fix the problem in the `KafkaRebalance` resource itself or with the overall Cruise Control deployment.
```

File: documentation/modules/cruise-control/proc-fixing-problems-with-kafkarebalance.adoc [hunk 2]
```diff
@@ -9,10 +9,12 @@
...
 Problems might include the following:
 
-* A misconfigured parameter.
+* A misconfigured parameter in the `KafkaRebalance` resource.
+* The `strimzi.io/cluster` label, in the `KafkaRebalance` resource, for specifying the Kafka cluster is missing.
```
Comment (PaulRMellor) [inline documentation/modules/cruise-control/proc-fixing-problems-with-kafkarebalance.adoc:16]:
```suggestion
* The `strimzi.io/cluster` label for specifying the Kafka cluster in the `KafkaRebalance` resource is missing.
```

File: documentation/modules/cruise-control/proc-fixing-problems-with-kafkarebalance.adoc [hunk 3]
```diff
@@ -9,10 +9,12 @@
...
+* The Cruise Control server is not deployed. The `cruiseControl` field, in the `Kafka` resource, is missing.
```
Comment (PaulRMellor) [inline documentation/modules/cruise-control/proc-fixing-problems-with-kafkarebalance.adoc:17]:
```suggestion
* The Cruise Control server is not deployed as the `cruiseControl` property in the `Kafka` resource is missing.
```

**Review by PaulRMellor — APPROVED** [review]
Couple suggestions, but looks good

**Pull request merged**